R/cov_ug.R

```diff
@@ -76,16 +77,54 @@ port <- function(N = 1, p = 3, d = 1, ug = NULL, zapzeros = TRUE) {
         sam[, , n] <- zapsmall(sam[, , n])
       }
     }
-  } else {
-    sam <- array(dim = c(p, p, N), data = runif(p * p * N))
+  }
+  return(sam)
+}
+
+
+#' @rdname cov_ug
+#' @useDynLib gmat
+#' @importFrom gRbase moralize
+#' @importFrom igraph as_adjacency_matrix
```

Comment:
This line and the above are not needed since igraph and gRbase are already imported as a whole for the package